crt_portal/cts_forms/templates/forms/report_details.html

```diff
@@ -12,11 +12,16 @@
     <div class="margin-bottom-1">
       {{ field }}
     </div>
+    <noscript>
+      Please enable JavaScript for the best experience using this website.
```

Comment:
The word counter functionality relies on JavaScript. As a future improvement, we could remove the initial word count HTML if JavaScript is not available, because it quickly becomes inaccurate. But as a minimum, this lets the user know that JavaScript is required for the best experience with the site.

Reply:
That makes sense